src/lib/containers/Login.tsx

```diff
@@ -199,14 +200,14 @@ class Login extends React.Component<Props, State> {
             onChange={this.handleChange}
           />
           {this.getLoginFailureView()}
-          <button
+          <Button
+            variant="primary"
             onSubmit={this.handleLogin}
             type="submit"
-            className="btn SRC-primary-background-color SRC-hoverWhiteText
-              SRC-whiteText m-1 SRC-google-button SRC-marginBottomTen"
+            className="SRC-google-button SRC-marginBottomTen"
```

Comment:
On top of this being less code/fewer CSS classes, the button also gets the default bootstrap hover behavior (it previously had no hover behavior).

Reply:
@nickgros 

The react-bootstrap dependency is actually for Bootstrap 4. If you ever run into any `react-bootstrap` tags that get styled weirdly, this is the reason.

`<Button>` will work because it happens to have the same `className` and HTML element type in both Bootstrap 3 and Bootstrap 4

I'm personally hesitant about adding more confusion to the Bootstrap 3/4 ambiguity (i.e. continuing using the Bootstrap 4 react library for more elements intended for Bootstrap 3 styling).

Reply:
This is a good point! I think we still get some benefit because we could still do `<button className="btn btn-primary"> ...` instead of `<button className="btn btn-primary SRC-primary-background-color otherStylesEtc"> ...`